main.py

```diff
@@ -28,43 +32,78 @@ def main():
         try: 
             if intent == Intents.GO:
                 pressReleaseKey(NoGbaMappings.A)
+
             elif intent == Intents.BACK:
                 pressReleaseKey(NoGbaMappings.B)
+                bagActions.markBagAsClosed()
```

Comment:
Instance of 'BagActions' has no 'markBagAsClosed' member :(
I don't know what to replace it with

Reply:
Ohhh, forgot to change that